AY2021S2-CS2103-T14-1/tp
# Implement Room CRUD features

# What this does

This PR implements the CRUD features for the Room object, inclusive of storage and UI changes. It adds a second window at bottom of the current UI to show operations conducted on the rooms. 

Part of #51
Closes #22, closes #29, closes #30, closes #31, closes #32, closes #33 

# How to test

1. Launch SunRez
2. Perform commands related to the room (See UG for guide)
3. SunRez should execute the commands related to the rooms.

# What is left to do

- Unit tests for all the new commands
- Additional unit tests for ModelManager 
- Integration tests for all the new commands
- Update UG
- Update DG
